docs/reference/schema.md

```diff
@@ -0,0 +1,539 @@
+        -Hazard process
+        -Exposure taxonomy
+        -Analytical  method 
+        -Applicability
+      }
+      class Loss{
+        -Hazard process
+        -Exposure taxonomy
+        -Loss frequency
+        -Loss metric
+      }          
+```
+
+## Dataset
+
+In addition to schema-specific attributes, each dataset is identified by a list of attributes based on the [Dublin Core Metadata Initiative Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms).
+
+```{jsonschema} ../../docs/_readthedocs/html/rdl_schema_0.1.json
+---
+collapse: spatial,resources
+addtargets:
+---
+```
+
+## Resource
+
+Other attributes are specific to individual resources, covering level of aggregation, resolution and format.
+
+```{jsonschema} ../../docs/_readthedocs/html/rdl_schema_0.1.json
+---
+pointer: /$defs/Resource
+collapse: temporal
+addtargets:
+---
+```
+
+## Hazard
+
+The hazard schema stores data about the intensity and occurrence probability of physical hazard phenomena such as floods, earthquakes, wildfires or others. The specific hazard process can be defined and measured with a specific intensity unit. For example, earthquake hazard may be represented as ground shaking, liquefaction or ground displacement.
+
+```{eval-rst}
+ .. mermaid::
+
+  classDiagram
+      Event set -- Event1
+      Event set -- Event2
+      Event set: Hazard type
+      Event set: Analytical method
+      class Event1{
+        Occurrence frequency
+        Time reference
+        Hazard trigger
+      }
+      class Event2{
+        Occurrence frequency
+        Time reference
+        Hazard trigger
+      }
+      Event1 -- Footprint1
+      Event1 -- Footprint2
+      Event2 -- Footprint3
+      Event2 -- Footprint4
+      class Footprint1{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+      class Footprint2{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+      class Footprint3{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+      class Footprint4{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+```
+
+The hazard schema specifies which type of analysis and data methodology that has generated the dataset. It supports either simulated probabilistic scenarios and empirical observations. If the dataset has been produced for a specific location, such a city, the name of the location can be included.
+
+When the scenario modelled refers to a specific period of time, this can be specified in terms of dates, period span and reference year. For example, an observed flood event that occurred from 1.10.2009 (time start) to 3.10.2009 (time end), spanning over 3 days (time span). When precise time collocation is unknown or not applicable, a general reference date such as "2009" is used to identify events (time year). This is also useful to specify future scenario, e.g. time year: 2050.
```

Comment:
```suggestion
When the scenario modelled refers to a specific period of time, this can be specified in terms of dates, period span and reference year. For example, an observed flood event that occurred from 2009-10-01 (time start) to 2009-10-03 (time end), spanning over 3 days (time span). When precise time collocation is unknown or not applicable, a general reference date such as "2009" is used to identify events (time year). This is also useful to specify future scenario, e.g. time year: 2050.
```